ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_numeric_dtypes),
+    dtype=st.sampled_from(ivy_np.valid_numeric_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.ceil"
+    ),
+)
+def test_numpy_ceil(
+    dtype_and_x,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    where = np_frontend_helpers.handle_where_and_array_bools(
```

Comment:
here ,  you should pass the input_dtype, as_variable and native_array in to handle_where_and_array_bools as lists, or this error happens:
input_dtype += ["bool"]
TypeError: can only concatenate str (not "list") to str